src/ChineseNumbers/SuzhouNumbers.h

```diff
@@ -0,0 +1,12 @@
+
+#ifndef FCITX5_MCBOPOMOFO_SUZHOUNUMBERS_H
+#define FCITX5_MCBOPOMOFO_SUZHOUNUMBERS_H
+
+#include <string>
+
+namespace ChineseNumbers::SuzhouNumbers {
```

Comment:
Same here. Prefer truly nested namespace declarations.

Reply:
Fixed in 92e46601d055fb57699128d5d2cb8634a1fef339